libcxx/test/std/input.output/filesystems/fs.op.funcs/fs.op.copy_file/copy_file_procfs.pass.cpp

```diff
@@ -0,0 +1,52 @@
+//===----------------------------------------------------------------------===//
+//
+// Part of the LLVM Project, under the Apache License v2.0 with LLVM Exceptions.
+// See https://llvm.org/LICENSE.txt for license information.
+// SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+//
+//===----------------------------------------------------------------------===//
+
+// UNSUPPORTED: c++03, c++11, c++14
+// REQUIRES: linux
```

Comment:
```suggestion
// REQUIRES: linux || freebsd
```

Since we use `copy_file_range` on FreeBSD, we should enable this test there.

Reply:
This test is for zero-length files in `/proc` specifically.

FreeBSD deprecated procfs a long while ago, and I don't know if it has any "virtual zero-length but not actually empty" files like linux does.

Reply:
Ack, disregard my comment then.